dotCMS/src/main/java/com/dotcms/storage/S3StoragePersistenceAPIImpl.java

```diff
@@ -0,0 +1,221 @@
+package com.dotcms.storage;
+
+public class S3StoragePersistenceAPIImpl {
+
+}
+/*
```

Comment:
why is this commented? if not used, it should be removed

Reply:
Sachez wanted to keep such code at hand but I guess you're right. Removd